gcp/modules/gcp-stackdriver-monitoring/notification_alerts_slack.tf

```diff
@@ -0,0 +1,12 @@
+resource "google_monitoring_notification_channel" "alerts_slack" {
```

Comment:
Recreating this would require manual action to authorize new channel with Slack.